src/Components/Admins/index.js

```diff
@@ -1,9 +1,33 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import Table from './Table';
 import styles from './admins.module.css';
 
 function Admins() {
+  const [admins, saveAdmins] = useState([]);
+
+  useEffect(async () => {
+    await fetch(`${process.env.REACT_APP_API_URL}/admins`)
+      .then((response) => response.json())
+      .then((response) => {
+        saveAdmins(response.data);
+      });
+  }, []);
+
+  const deleteAdmin = async (id) => {
+    await fetch(`${process.env.REACT_APP_API_URL}/admins/${id}`, {
+      method: 'DELETE'
+    });
+    saveAdmins([...admins.filter((newListItem) => newListItem._id !== id)]);
+  };
+
   return (
     <section className={styles.container}>
       <h2>Admins</h2>
+      <Table admins={admins} deleteAdmin={deleteAdmin} />
+      <a href={`admins/form`}>
```

Comment:
```suggestion
      <a href={'admins/form'}>
```
Remove backsticks (`) as you are not using a template literal in this line

Reply:
`     <a href={'admins/form'}>
`
done